configure.ac

```diff
@@ -1938,6 +1938,13 @@ if test "$Py_BOLT" = 'true' ; then
   DEF_MAKE_ALL_RULE="bolt-opt"
   DEF_MAKE_RULE="build_all"
 
+  AC_SUBST(READELF)
+  AC_CHECK_TOOLS(READELF, [readelf], "notfound")
+  if test "$READELF" == "notfound"
+  then
+    AC_MSG_ERROR([readelf is required for a --enable-bolt build but could not be found.])
+  fi
```

Comment:
This can fail even if `--enable-bolt` is not passed to `configure`. I'd suggest to only run this block if `--enable-bolt` was passed.

Reply:
@erlend-aasland It's already checked from the L1936 :)
````

if test "$Py_BOLT" = 'true' ; then
  PREBOLT_RULE="${DEF_MAKE_ALL_RULE}"
  DEF_MAKE_ALL_RULE="bolt-opt"
  DEF_MAKE_RULE="build_all"

  AC_SUBST(READELF)
  AC_CHECK_TOOLS(READELF, [readelf], "notfound")
  if test "$READELF" == "notfound"
  then
    AC_MSG_ERROR([readelf is required for a --enable-bolt build but could not be found.])
  fi
````

Reply:
Oh my, sorry for not noticing that! :)